package.json

```diff
@@ -65,11 +67,12 @@
     "mocha": "^8.3.2",
     "mocha-lcov-reporter": "^1.3.0",
     "node-forge": "^0.10.0",
+    "npm": "^7.8.0",
```

Comment:
This looks suspicious to me. Can this be removed, why is this here?

Reply:
Thanks for catching this. It was a typo, I've removed it.